src/types/QueryFilterTypes.ts

```diff
@@ -121,3 +124,9 @@ export interface UnknownFilter extends Filter {
 export interface TautologyFilter extends Filter {
   type: 'truth';
 }
+
+export interface CQLInterval {
```

Comment:
Might suggest renaming this to not have it appear that we are defining a specific CQL type. Technically, this is a construct we made from the results of QueryFilterParsing, so it slightly deviates from cql system types.

What about, `ParsedFilterInterval` or someth. @hossenlopp ?

Reply:
I think `ParsedFilterInterval` is okay. Definitely want to differentiate it from the CQL system types.